index: Add platforms description and logo (#8)
hostmobility/documentation: docs/index.md

title: Index
---

![Host Mobility](https://hostmobility-eng.setek.se/wp-content/uploads/2021/06/logo-host-mobility200.png)

## The Host Mobility Platforms

Host Mobility designs and manufactures small computing devices that can be connected to vehicles of any kind, fetch diagnostics from a long range of data buses, register the current location, transmit data for further analysis, and much more – all using standard Linux APIs. You are free to run any application on top of our hardware platforms.

Host Mobility provides a complete Linux platform with driver support for all hardware interfaces and with a customizable distribution.

> All hardware interfaces are accessible via well defined APIs. We use the the standard Linux way of doing things as much as possible. This way the platform environment is familiar to developers who has worked with embedded Linux in their past.

Reviewer: ```suggestion
All hardware interfaces are accessible via well-defined APIs. We prefer the standard Linux way of doing things as much as possible. This way the platform environment is familiar to developers who have worked with embedded Linux in the past.
```

Reviewer 2: the text is removed is it as it should @mattiasbus ?

| Name                                      | Description  |
| -----                                     | ------------ |
| [Host Monitor X](hardware/host-monitor-x) |              |

> | [MX4-T30FR](hardware/mx-4-t30fr)           |              |

Reviewer 2: where is the hardware folder?

Reviewer 2: found it on #9

title: Index
---

> ![Host Mobility](https://hostmobility-eng.setek.se/wp-content/uploads/2021/06/logo-host-mobility200.png)

Reviewer 2: only issue is this logo. should be replaced
